lib/Echidna/Solidity.hs

```diff
@@ -108,6 +108,15 @@ data SolConf = SolConf { _contractAddr    :: Addr             -- ^ Contract addr
                        }
 makeLenses ''SolConf
 
+readSolcs :: FilePath -> IO (Maybe (Map Text SolcContract, SourceCache))
```

Comment:
probably want to rename this, like `readSolcBatch` or something

Reply:
Done.